package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "labeler",
-  "version": "3.0.0",
+  "version": "3.0.1",
```

Comment:
do we need to bump this in this PR?

Reply:
I can bump it in a subsequent commit if that's our workflow—as part of my cleanup in this repo I'll add guidelines on our release process too, since there aren't any right now.

Reply:
I would suggest not bump it in this PR, I think we normally bump it when we are ready to release a newer version.